test/unit/nedap/speced/def/spec_assertion.cljc

```diff
@@ -53,3 +54,23 @@
   (is (spec-assertion-thrown? 'number? (returns-number-fn "1234")))
   (is (spec-assertion-thrown? 'string? (accepts-string (->ExampleRecord) 1234)))
   (is (spec-assertion-thrown? 'string? (returns-string (->ExampleRecord) 1234))))
+
+(defn parameterized-defn [spec x]
+  (check! spec x)
+  x)
+
+#?(:clj (defmacro when-not-u-s-1-0-0
+          {:style/indent 0}
+          [& body]
+          (when-not (-> "nedap.speced.def.testing.utils-spec-dep" System/getProperty #{"1.0.0"})
+            `(do
+               ~@body))))
```

Comment:
I'm not sure we want to check for 1.0.0 exactly. what about <1.2.0 ? because 1.1.0 is also legacy, and 0.9 is as well.

Reply:
The point is about testing the last version exhibiting the old behavior; not testing every version. They are certainly identical for the property being tested.